infra/templates/azure-simple/appdev.tf

```diff
@@ -1,18 +1,12 @@
 locals {
-  service_plan_name   = "${var.name}-sp"
-  app_insights_name   = "${var.name}-ai"
-  resource_group_name = "${var.name}-rg"
+  service_plan_name = "${var.name}-sp"
```

Comment:
[nit] abbreviations between 1-3 letters are a bit difficult to reason about. Is it worthwhile to expand them. e.g. `sp` -> `svcplan` or something?